<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorAutoSaveWidget</class>
 <widget class="QWidget" name="EditorAutoSaveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkAutoBackupEditContents">
     <property name="text">
      <string>Auto backup editing contents</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpEnableAutoSave">
     <property name="title">
      <string>Enable auto save</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Time interval:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinInterval">
           <property name="suffix">
            <string>minutes</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Objects to save</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rbCurrentFile">
           <property name="text">
            <string>Current File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAllOpennedFiles">
           <property name="text">
            <string>All files opened</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbProjectFiles">
           <property name="text">
            <string>Project files</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Save strategy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="rbOverwrite">
           <property name="text">
            <string>Overwrite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAppendUNIXTimestamp">
           <property name="text">
            <string>Append UNIX timestamp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAppendFormattedTimestamp">
           <property name="text">
            <string>Append formatted timestamp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFilename">
        <property name="text">
         <string>Demo file name:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
